packages/util-rlp/src/decoder/listShort.ts

```diff
@@ -5,9 +5,9 @@
 import { DecodeFunc, DecodeOutput } from './types';
 
 export default function decodeListShort (decode: DecodeFunc, input: Uint8Array): DecodeOutput {
+  const decoded: Array<any | Uint8Array> = [];
```

Comment:
This is not "quite" correct - any includes Uint8Array anyway. But it gives a tiny bit of intent. (RLP will be removed shortly as it stands, PoC-3 have gotten rid of it completely).

(Funnily this was not one of the issues raised, they are all in trie-hash, however the same issue would manifest here)